addons/website/static/src/scss/website.scss

```diff
@@ -5,10 +5,15 @@
 $-seen-urls: ();
 @each $alias, $key in $o-font-aliases-to-keys {
     $-url: o-get-font-info($alias, 'url');
+    $-attachment: o-get-font-info($alias, 'attachment');
+    $-name: o-get-font-info($alias, 'name');
     @if $-url and index($-seen-urls, $-url) == null {
         $-seen-urls: append($-seen-urls, $-url);
         @import url("https://fonts.googleapis.com/css?family=#{unquote($-url)}&display=swap");
     }
+    @if $-attachment {
+        @import url("/web/content/#{$-attachment}/google-font-#{unquote($-name)}");
+    }
```

Comment:
Maybe `else if `instead of two `if ` with the calls to `o-get-font-info` inside the else if ?

Reply:
Sure :+1: